grafana/grafana
# Elasticsearch: Run requestAllIndices trough resource call if enableElasticsearchBackendQuerying enabled

In this PR, we are use ES backend's `CallResource`  to run `requestAllIndices` trough resource call if `enableElasticsearchBackendQuerying` is enabled.

Part of #54011

How to test this:
1. Run `make devenv sources=elastic`
1. Make sure that `enableElasticsearchBackendQuerying` is enabled
1. Open ES in Explore, choose `Average` metric and click on `Select Field`
1. You should see available fields
1. Open network tab and inspect if `_mapping` goes trough resources - e.g. `/resources/logs-2023.05.04/_mapping`

<img width="1488" alt="image" src="https://user-images.githubusercontent.com/30407135/236187507-f6772c65-9100-4fb4-97fa-0f079e400bdc.png">
<img width="671" alt="image" src="https://user-images.githubusercontent.com/30407135/236187533-6afe102a-6335-402b-9d00-8c63f29a422d.png">
